### DISCUSSION TITLE
Meek Azure Support?

### DISCUSSION THREAD

**Discussion by CEOXeon** [guardianproject/orbot-android#1196] [Q&A] [answered]
Is it possible to connect using a meek bridge like in the Tor Browser.

![Screenshot_20241101-120957_Orbot.jpg](https://github.com/user-attachments/assets/bdd39d3f-c09f-4ba8-9886-0f9266b1f47c)

![Screenshot_20241101-121046_Tor Browser.jpg](https://github.com/user-attachments/assets/68efcd09-9e33-433a-8fdc-cdf4eb6edbb0)

**Comment by n8fr8** [comment, marked as answer]
We were told by the Tor team that Meek Azure was being deprecated. If it is still available in TBA, then I guess it isn't. We used to support.

Anyhow, we will be doing a release soon with Web Tunnel support, and will see if we can add Meek back in.

**Reply by CEOXeon** [reply]
The thing is the that I'm currently in a network that is so heavily censored that I can only connect using meek azure. Even snowflake can not bypass only meek azure. Hope that webtunnel has the same or better effect like meek.
